src/components/FilmDetail.vue

```diff
@@ -46,29 +68,24 @@ export default {
     }
   },
 
-  data () {
-    return {
-      info: {},
-      URL_IMG,
-      IMG_SIZE_LARGE,
-      errored: false,
-    }
-  },
+  created () {
+    const urlDetail = `${URL_DETAIL}${this.id}?api_key=${API_KEY}`
+    const urlTrailers = `${URL_DETAIL}${this.id}${URL_VIDEO}?api_key=${API_KEY}`
+    const urlCast = `${URL_DETAIL}${this.id}/credits?api_key=${API_KEY}`
+    const urlRecommendations = `${URL_DETAIL}${this.id}/recommendations?api_key=${API_KEY}`
 
-  mounted () {
-    axios
-      .get(`${URL_DETAIL}${this.id}?api_key=${API_KEY}`)
-      .then(response => {
-        this.info = response.data
-      })
-      .catch(error => {
-        console.log(error)
-        this.errored = true
-      })
-      .finally(() => {
-        this.asyncDataStatusFetched()
-      })
-  }
+    this.fetchDetailFilm({ urlDetail })
+      .then(() => this.asyncDataStatusFetched())
```

Comment:
đoạn này a có thể viết gọn lại thành .then(this.asyncDataStatusFetched)

Reply:
ok e